### DISCUSSION TITLE
Can I disable preview of a source URL when I hover mouse over an article

### DISCUSSION THREAD

**Discussion by igor17721** [FreshRSS/FreshRSS#5880] [Q&A]
Is it possible to make a link not to show at the bottom when I hover mouse over articles inside my feed, so it would show only when I expand an article and hover over the title? Or it would be even better to completely disable it and somehow make the original link to appear at the bottom of an article. 

![Untitled2](https://github.com/FreshRSS/FreshRSS/assets/108469300/9e9c234e-b7fc-4dd7-b68f-07b631103dbb)

**Comment by Alkarex** [comment]
When you mouse over article titles, it is the original link that appears at the bottom of your Web browser.
There is no option to control that in FreshRSS, as it is left to the default behaviour of your Web browser.
Alternatively, you can use keyboard shortcuts such as `n` or `j` instead of mouse
